tensorflow_datasets/image/lfw.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import os
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+_URL="http://vis-www.cs.umass.edu/lfw/lfw.tgz"
+LFW_IMAGE_SHAPE=(250,250,3)
+LFW_CITATION="""@TechReport{LFWTech,
+  author = {Gary B. Huang and Manu Ramesh and Tamara Berg and 
+                  Erik Learned-Miller},
+  title = {Labeled Faces in the Wild: A Database for Studying 
+                  Face Recognition in Unconstrained Environments},
+  institution = {University of Massachusetts, Amherst},
+  year = 2007,
+  number = {07-49},
+  month = {October}}"""
+
+
+class LFW(tfds.core.GeneratorBasedBuilder):
+    URL="http://vis-www.cs.umass.edu/lfw/#resources"
+    
+    VERSION = tfds.core.Version("2.0.0")
+    def _info(self):
+        return tfds.core.DatasetInfo(
+            builder=self,
+            description=("Labeled Faces in the Wild: A Database for Studying Face Recognition in Unconstrained Environments"),
+            features=tfds.features.FeaturesDict({
+                "anchor": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
+                "positive": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
+                "negative": tfds.features.Image(shape=LFW_IMAGE_SHAPE),
+            }),
+            urls=[self.URL],
+            citation=LFW_CITATION,
+        )
+    
```

Comment:
You should run `pylint` to cleanup unnecessary empty lines. You can check this [link](https://www.tensorflow.org/community/contribute/code_style) about code style.